service/src/lib.rs

```diff
@@ -46,10 +43,10 @@ pub mod chain_spec;
 
 use std::sync::Arc;
 use std::time::Duration;
-use polkadot_primitives::{parachain, AccountId, Block, Hash, BlockId};
+use polkadot_primitives::{parachain, parachain::CollatorId, Block, Hash, BlockId};
 use polkadot_runtime::{GenesisConfig, RuntimeApi};
 use polkadot_network::gossip::{self as network_gossip, Known};
-use primitives::ed25519;
+use primitives::{Pair, ed25519};
```

Comment:
i'm surprised that `Pair` would be exported at the root if there is more than one kind of crypto. for readability we should explicitly import from some submodule. I have no idea what kind of crypto this keypair is for.

Reply:
It's the trait.